schema_version: 1.2.0
id: CVE-2017-16667
aliases: []
modified: 2019-04-30T16:07Z
published: 2017-11-08T18:29Z
details: backintime (aka Back in Time) before 1.1.24 did improper escaping/quoting
  of file paths used as arguments to the 'notify-send' command, leading to some parts
  of file paths being executed as shell commands within an os.system call in qt4/plugins/notifyplugin.py.
  This could allow an attacker to craft an unreadable file with a specific name to
  run arbitrary shell commands.
severity:
- type: CVSS_V3
  score: 7.8
- type: CVSS_V2
  score: 9.3
affected:
- package:
    name: backintime
    cpe: cpe:2.3:a:backintime_project:backintime
- ranges:
  - type: GIT
    repo: https://github.com/bit-team/backintime
    events:
    - introduced: '0'
    - fixed: cef81d0da93ff601252607df3db1a48f7f6f01b3
- database_specific:
    CWE:
    - CWE-78
    CVSS:
      Score: 7.8
      Severity: HIGH
      Code: CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H
references:
- type: FIX
  url: https://github.com/bit-team/backintime/commit/cef81d0da93ff601252607df3db1a48f7f6f01b3
- type: WEB
  url: https://github.com/bit-team/backintime/releases/tag/v1.1.24
- type: WEB
  url: https://security.gentoo.org/glsa/201801-06
- type: REPORT
  url: https://github.com/bit-team/backintime/issues/834
